tests/digital_identity_service/policy/wanted.attribute.builder.spec.js

```diff
@@ -131,4 +131,48 @@ describe('WantedAttributeBuilder', () => {
     expect(wantedAttribute).toBeInstanceOf(WantedAttribute);
     expect(JSON.stringify(wantedAttribute)).toBe(expectedJson);
   });
+
+  it('should build a wanted attribute with alternative name', () => {
+    const wantedAttribute = new WantedAttributeBuilder()
+      .withName(TEST_NAME)
+      .withAlternativeName(`alt-${TEST_NAME}`)
+      .build();
+
+    const expectedJson = JSON.stringify({
+      name: TEST_NAME,
+      optional: false,
+      alternative_names: [`alt-${TEST_NAME}`],
+    });
+    expect(wantedAttribute).toBeInstanceOf(WantedAttribute);
+    expect(JSON.stringify(wantedAttribute)).toBe(expectedJson);
+  });
+
+  it('should build a wanted attribute with alternative names', () => {
+    const wantedAttribute = new WantedAttributeBuilder()
+      .withName(TEST_NAME)
+      .withAlternativeNames([`${TEST_NAME}-alt1`, `${TEST_NAME}-alt2`])
+      .build();
+
```

Comment:
There is still the case of calling `withAlternativeNames` first and then calling `withAlternativeName` after for once or more.

And also the opposite which will only contain the last value if `withAlternativeNames` was called last.

Reply:
Ok, added the first case to the tests.